packages/react/src/RelativeTime/RelativeTime.tsx

```diff
@@ -6,14 +6,14 @@ import {createComponent} from '../utils/create-component'
 const RelativeTimeComponent = createComponent(RelativeTimeElement, 'relative-time')
 
 const localeOptions: Intl.DateTimeFormatOptions = {month: 'short', day: 'numeric', year: 'numeric'}
-function RelativeTime({date, datetime, children, ...props}: RelativeTimeProps) {
+function RelativeTime({date, datetime, children, noTitle, ...props}: RelativeTimeProps) {
   if (datetime) date = new Date(datetime)
   return (
-    <RelativeTimeComponent {...props} date={date}>
+    <RelativeTimeComponent {...props} date={date} no-title={noTitle ? '' : undefined}>
       {children || date?.toLocaleDateString('en', localeOptions) || ''}
     </RelativeTimeComponent>
   )
 }
 
-export type RelativeTimeProps = ComponentProps<typeof RelativeTimeComponent>
+export type RelativeTimeProps = ComponentProps<typeof RelativeTimeComponent> & {noTitle?: boolean}
```

Comment:
I think this is no longer necessary, now that `@github/relative-time-element` has been updated. At least, when I remove it in my codespace, I don’t get type errors.

```suggestion
export type RelativeTimeProps = ComponentProps<typeof RelativeTimeComponent>
```

Reply:
Ah, that makes sense! Thanks!